js/data.js

```diff
@@ -13,3 +13,25 @@ function logBefore(event) {
   var entriesData = JSON.stringify(data);
   localStorage.setItem('entries-data', entriesData);
 }
+
+window.addEventListener('DOMContentLoaded', appendDOM)
+
+var leftoverEntries = localStorage.getItem('entries-data');
+if (leftoverEntries !== null) {
+  data = JSON.parse(leftoverEntries);
+}
+
+function appendDOM() {
+  for (var i = 0; i < data.entries.length; i++) {
+    entryDOM(data.entries[i]);
+  }
+  if (data.view === 'entry-form'){
+      $newEntry.className = '';
+      $entriesPage.className = 'hidden';
+  } else {
+      $newEntry.className = 'hidden';
+      $entriesPage.className = '';
+  }
+}
```

Comment:
You should be doing this in `main.js`, it doesn't have anything to do with the data model or localStorage, which is all that should be in `data.js`